Tests/WorldCup/ScoreBoardTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Cheba\PhpUnit\Tests\WorldCup;
+
+use Cheba\PhpUnit\WorldCup\ScoreBoard;
+use PHPUnit\Framework\TestCase;
+
+class ScoreBoardTest extends TestCase
+{
+    public function testStartGame()
+    {
+        $scoreBoard = new ScoreBoard();
+        $game = $scoreBoard->startGame('Mexico', 'Canada');
+
+        $this->assertEquals('Mexico', $game->homeTeam);
+        $this->assertEquals('Canada', $game->awayTeam);
+        $this->assertEquals(0, $game->homeScore);
+        $this->assertEquals(0, $game->awayScore);
+
+    }
+
+    public function testUpdateScore()
+    {
+        $scoreBoard = new ScoreBoard();
+        $game = $scoreBoard->startGame('Mexico', 'Canada');
+
+        $scoreBoard->updateScore($game, 1, 2);
+
+        $this->assertEquals(1, $game->homeScore);
+        $this->assertEquals(2, $game->awayScore);
+    }
+
+    public function testFinishGame()
```

Comment:
This test always works because it never saves any matched from the beginning, and that test always passes. You never tested the summary for an existing match.

Reply:
Rewrite the method, added a check for game completion and  and matches.